packages/material-ui-codemod/README.md

```diff
@@ -61,7 +61,7 @@ npx @material-ui/codemod@next <transform> <path> --jscodeshift="--printOptions='
 
 #### 🚀 `preset-safe`
 
-A combination of all important transformers for migrating v4 to v5. ⚠️ This mod is not idempotent (`theme-breakpoints`) and should be run only once.
+A combination of all important transformers for migrating v4 to v5. ⚠️ This codemod is not idempotent (`theme-breakpoints`) and should be run only once.
```

Comment:
```suggestion
A combination of all important transformers for migrating v4 to v5. ⚠️ This codemod should be run only once.
```
Let's just get rid of this term. It's unnecessary jargon and can cause confusion if it's not actually correct.